{"tx_status": "REJECTED", "tx_failure_reason": {"code": "TRANSACTION_FAILED", "error_message": "Error at pc=0:15:\nGot an exception while executing a hint.\nCairo traceback (most recent call last):\nUnknown location (pc=0:527)\nUnknown location (pc=0:515)\nUnknown location (pc=0:471)\nUnknown location (pc=0:105)\n\nError in the called contract (0x1fc60ff13e22ce69694ca292d3c256028205aec5b7742013eb1cc2cd7e0a26e):\nError at pc=0:816:\nAn ASSERT_EQ instruction failed: 0 != 1.\nCairo traceback (most recent call last):\nUnknown location (pc=0:1005)\nUnknown location (pc=0:931)"}}